cmd/stat.go

```diff
@@ -89,7 +89,7 @@ func (c statMessage) JSON() string {
 }
 
 // parseStat parses client Content container into statMessage struct.
-func parseStat(targetAlias string, c *clientContent) statMessage {
+func parseStat(c *clientContent) statMessage {
 	content := statMessage{}
 	content.Date = c.Time.Local()
 	// guess file type.
```

Comment:
with your change, stat listing shows an extra slash before the object name. Needs to be fixed.
example
```
Name      : /bucket1/sas/b/v/c
Date      : 2019-02-13 23:25:58 PST 
Size      : 26B    
ETag      : 38103f9a76bb14e2abbc051b65722cff 
Type      : file 
```

Reply:
> with your change, stat listing shows an extra slash before the object name. Needs to be fixed.

Not able to find any extra slash, can you reproduce this again ? and tell me how to do that ?

By the way, there is another change in stat command, it won't stat multiple objects unless '-r' is specified. I just wanted to avoid to duplicate code because stat and rm have similar needs:  Stat() but avoid failiure in case of encryption.

Reply:
this is happening for stat of encrypted objects without encryption key - master shows object name as object, your patch shows /bucket/object

Reply:
> this is happening for stat of encrypted objects without encryption key - master shows object name as object, your patch shows /bucket/object

Fixed